vendor/foxy/include/foxy/impl/session/async_read.impl.hpp

```diff
@@ -1,41 +1,66 @@
 //
```

Comment:
The majority of changes in these files is auto formatting, I only added an async_read_some wrapper.

Ideally this change would take place in our fork.

Reply:
I think it is important not to auto format any of this code. It will make reconciling it with anything very annoying.

Reply:
Great point. Fixed.